1.8/administration/access-node/tunnel.md

```diff
@@ -114,16 +114,33 @@ DC/OS Tunnel provides you with full access to the DNS, masters, and agents from
 * [SSH access](/docs/1.8/administration/access-node/sshcluster/) (key authentication only).
 * [The OpenVPN client](https://openvpn.net/index.php/open-source/downloads.html) for VPN functionality.
 
-**Note:** Only Linux and OS X are supported currently.
+**Note:** Only Linux and macOS are supported currently.
```

Comment:
Do you mean give people a command to run for installing the CLI, SSH access, and OpenVPN client? Besides installing the `tunnel-cli` module, they'll each require a tutorial of their own